pkgs/top-level/perl-packages.nix

```diff
@@ -7119,6 +7119,20 @@ let
     };
   };
 
+  FileGrep = buildPerlPackage {
+    pname = "File-Grep";
+    version = "0.02";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/M/MN/MNEYLON/File-Grep-0.02.tar.gz";
+      sha256 = "462e15274eb6278521407ea302d9eea7252cd44cab2382871f7de833d5f85632";
+    };
+    meta = {
+      description = "Find matches to a pattern in a series of files and related functions";
+      maintainers = [ maintainers.limeytexan ];
+      # license unknown
```

Comment:
```suggestion

```

`meta` is bot-maintained so it will be filled (or left unfilled) on the next bot run based on the CPAN info